tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -191,43 +197,91 @@ def test_positive_update_katello_certs(self):
                 assert result.return_code == 0, 'Not all services are running'
 
     @destructive
-    @stubbed()
-    def test_positive_update_capsule_certs_using_absolute_path(self):
-        """Update certificates on a currently running internal capsule.
+    def test_positive_generate_capsule_certs_using_absolute_path(self):
+        """Create Capsule certs using absolute paths.
 
         :id: 72024757-be6f-49f0-8b88-c57c83f5e7e9
 
         :steps:
 
-            1. Generate custom certs
-            2. Run katello-certs-check with the required valid arguments
-            3. Run  capsule-certs-generate with custom certs and absolute path
+            1. Use Certificates generated in class setup
+            2. Run capsule-certs-generate with custom certs and absolute path
                for --certs-tar
+            3. Assert the certs tar file was created.
 
-        :expectedresults: Katello-certs should be updated.
+        :expectedresults: Capsule certs are generated.
 
-        :CaseAutomation: notautomated
+        :BZ: 1466688
+
+        :CaseAutomation: automated
         """
+        with get_connection(timeout=300) as connection:
+            connection.run('mkdir /root/capsule_cert 2>/dev/null')
```

Comment:
why the output to /dev/null?

Reply:
wanted to suppress errors if directory exists, in case tests are restarted. Will use the shorter and possibly safer "-p" option.
Thank you